config/site/src/_includes/success_stories.html

```diff
@@ -19,42 +19,75 @@ <h3 class="text-3xl font-semibold mb-6">
         <div class="flex flex-col md:flex-row justify-between gap-6">
           <!-- Ingestion Performance Card -->
           <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg md:w-1/3">
-            <h4 class="text-xl font-bold mb-4">Ingestion Latency</h4>
+            <h4 class="text-xl font-bold mb-4">Ingestion</h4>
             <p
               class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mb-2">
-              &lt; 400ms</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">p99 latency</p>
+              &lt; 400ms
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              p99 latency
+            </p>
             <p
               class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
-              &lt; 1s</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">p99.99 latency
+              &lt; 1s
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              p99.99 latency
+            </p>
+            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
+              200k/s
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              documents/s ingested
             </p>
           </div>
           <!-- GraphQL API Latency Card -->
           <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg md:w-1/3">
-            <h4 class="text-xl font-bold mb-4">Query Latency</h4>
+            <h4 class="text-xl font-bold mb-4">Query</h4>
             <p
               class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mb-2">
-              &lt; 350ms</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">p99 latency</p>
+              &lt; 350ms
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              p99 latency
+            </p>
             <p
               class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
-              &lt; 2.5s</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">p99.99 latency
+              &lt; 2.5s
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              p99.99 latency
+            </p>
+            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
+              500 QPS
```

Comment:
We go over 500 QPS on a regular basis and can handle more than that--so maybe say `> 500 QPS`?